lib/assessment.rb

```diff
@@ -13,28 +18,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce do |item, memo|
+      memo += item
```

Comment:
`reduce` and `+` are the right tools. However, `+=` is not necessary, because the value that gets put into `memo` the next time around is what the block _returns_, and both `memo + item` and `memo += item` will return the same value.